centrifuge/api.txt

```diff
@@ -0,0 +1,373 @@
+// Signature format: 4.0
```

Comment:
This file contains public API of the library, with `internal` package excluded.

We assume `internal` package is not meant to be used by consumers of the library, thus don't care about compatibility changes in the internal package.